packages/synapse-react-client/src/style/components/_submission-page.scss

```diff
@@ -17,6 +17,8 @@
     display: flex;
     gap: 10px;
     button {
+      font-size: 16px;
+      color: white;
```

Comment:
Maintain styling of Submission page buttons -- MUI button defaults to black text color for success button and 14px font size for both buttons